poker-pals/src/components/pokerComponents/ReportPocket.vue

```diff
@@ -241,7 +241,8 @@ export default {
             }
             else {
                 // Send a filled report to the parent for submission to server
-                this.$parent.submitReport(this.selected, this.message, this.report_OffenderName, this.report_OffenderMessageId);
+                let submittingUser = this.$parent.userID;
```

Comment:
Where are you getting this id from? Poker component isn't passing userID though

Reply:
this.$parent.userID accesses the Poker component and grabs the id of the current user.

Reply:
Yeaaaah, this has changed in my PR

Reply:
You might need to do `this.$parent.userData.id` depending on who merges first